app/api/cruds/weekday_crud.py

```diff
@@ -0,0 +1,82 @@
+from django.core.exceptions import ObjectDoesNotExist, ValidationError
+
+import graphene
+from graphene_django import DjangoObjectType
+from graphql_relay.node.node import from_global_id
+
+from app.timetables.models import Weekday
+from .utils import get_errors
+
+
+class WeekdayNode(DjangoObjectType):
+    original_id = graphene.Int()
+
+    class Meta:
+        model = Weekday
+        filter_fields = {
+            'name': ['icontains']
+        }
+        filter_order_by = ['name', '-name']
+        interfaces = (graphene.relay.Node,)
+
+    def resolve_original_id(self, args, context, info):
+        return self.id
+
+
+class CreateWeekday(graphene.relay.ClientIDMutation):
+    weekday = graphene.Field(WeekdayNode)
+    errors = graphene.List(graphene.String)
+
+    class Input:
+        name = graphene.String(required=True)
+
+    @classmethod
+    def mutate_and_get_payload(cls, input, context, info):
+        try:
+            weekday = Weekday(
+                name=input.get('name')
+            )
+            weekday.full_clean()
+            weekday.save()
+            return CreateWeekday(weekday=weekday)
```

Comment:
Though it works fine as it is, `cls(weekday=weekday)` in place of `CreateWeekday(weekday=weekday)` is better off here and everywhere else with `@classmethod` decorator. That way, change in the name of the class wouldn't require change to those places.

If you agree with this, please extend the change to the graphql implementation for `user` and `dish` models. I know it's outside this story - it helps prevent technical debt from accruing. Thanks.